packages/shared/providers.yaml

```diff
@@ -787,6 +787,27 @@ calendly:
         - owner
     docs: https://docs.nango.dev/integrations/all/calendly
 
+brightcrowd:
+    display_name: BrightCrowd
+    categories:
+        - alumni-data
+    auth_mode: BASIC
+    token_url: https://bcb-staging.auth.us-east-1.amazoncognito.com/oauth2/token
+    token_request_auth_method: basic
+    token_params:
+        grant_type: client_credentials
+        scope: bcb.partner/book.read
```

Comment:
Not sure it's correct, you have to request this scope in your own integration no?